packages/php-db-import-export/phpstan-baseline.neon

```diff
@@ -1,5 +1,15 @@
 parameters:
 	ignoreErrors:
+		-
+			message: "#^Method Keboola\\\\Db\\\\ImportExport\\\\Backend\\\\Bigquery\\\\BigqueryException\\:\\:createExceptionFromJobResult\\(\\) has parameter \\$jobInfo with no value type specified in iterable type array\\.$#"
+			count: 1
+			path: src/Backend/Bigquery/BigqueryException.php
+
+		-
+			message: "#^Method Keboola\\\\Db\\\\ImportExport\\\\Backend\\\\Bigquery\\\\BigqueryException\\:\\:getErrorMessageForErrorList\\(\\) has parameter \\$parsingErrors with no value type specified in iterable type array\\.$#"
+			count: 1
+			path: src/Backend/Bigquery/BigqueryException.php
+
```

Comment:
:-(

Reply:
to je update stanu enom sa to přesunulo

Reply:
já vím, ale tohle je zrovna chyba, která se na 90% dá jednoduše fixnout

Reply:
jo to určitě enom toto není to pravé místo, většina těch chyb ide určitě fixnůt